compute_endpoint/globus_compute_endpoint/endpoint/utils/__init__.py

```diff
@@ -75,6 +82,11 @@ def _redact_url_creds(raw: _T, redact_user=True, repl="***", count=0) -> _T:
 
 
 def is_privileged(posix_user=None):
+    if not has_pyprctl:
```

Comment:
I wonder if this can be combined with the check in endpoint_manager.  Or, at least use the same message in cli.py and endpoint_manager.py regarding not supported on this system.

3 checks in 3 different files makes it harder to remember where to modify/add to later.

Reply:
I'm fine to let you approve.  Please finish this conversation as you see fit.

Reply:
Good point, this probably isn't necessary with the cli and endpoint manager checks, I'll remove it.

Also not a huge fan of the duplication between cli and endpoint_manager, so I'll remove the `"cannot run multitenant endpoints: ` part of the endpoint_manager message.

Either way, the message you get when you try to run an MT EP on macOS is the top-level one, ie:

```
$ gce start local-mt
Error: multi-tenant endpoints are not supported on this system
```